app/models/site_config.rb

```diff
@@ -72,6 +72,12 @@ class SiteConfig < RailsSettings::Base
   field :suggested_tags, type: :array, default: %w[beginners career computerscience javascript security ruby rails swift kotlin]
   field :sidebar_tags, type: :array, default: %w[help challenge discuss explainlikeimfive meta watercooler]
 
+  # Default ux config
+  field :feed_style, type: :string, default: "base" # Should the feed show images?
```

Comment:
I really wish `rails-settings-cached` would support some sort of enum, as string keys make it hard to figure out all the available options (it doesn't, I checked the source). 

We should see if this gem really will be able to support our generalization needs and make a call at the end of this cycle.